### PR TITLE
Makes mechs unable to face the opposite direction instantly. 

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request

Makes mechs unable to face the opposite direction instantly. Instead they will pick either left or right and turn there first before fully turning around.

## Why It's Good For The Game
Mechs have multiple mechanics relating to direction (increased damage, cant shoot, etc) but can just negate these quickly due to just being able to flip around, this fixes that.

## Changelog
:cl:
balance: mechs can no longer flip directions instantly
/:cl:

### CODE REVIEW

**Comment by Sealed101** [comment]
it would be fine if the direction the mech chooses to turn into was predictable, but leaving it up to rng is shabby imo

**Comment by Fikou** [comment]
submit some fixes to get to at least 0 gbp before making balance changes

**Comment by Watermelon914** [comment]
I don't like this change,

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
Can you make it less random and just change it to a time-based delay instead?

**Comment by moocowswag** [comment]
I suggest only doing this to slower mechs like the durand due to its really strong punches that can instantly win a fight in melee against nonhulk.

On other mechs, in melee combat this vastly reduces the skill ceiling of the pilot who needs to constantly turn and land hit after hit after hit against an enemy that can constantly strafe back.

Tldr this is justified on durand and might have a more severe impact on other other mechs that are more reliant on their guns in melee.

**Comment by Mothblocks** [comment]
rip pariah having negative a billion gbp

**Comment by castawaynont** [comment]
> You currently have a negative Fix/Feature pull request delta of -358. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

how

**Comment by TheBoondock** [comment]
There goes the GBP debt, he can't get away with this 😔

**Comment by Mothblocks** [comment]
me making a comment and immediately marking myself as spam was supposed to clue everyone in

**Comment by Jackal-boop** [comment]
Nukie mechs are already a huge gamble. This will make them unusable

**Comment by theta060** [comment]
ahh yes nerf mechs because they weren't already lacking
